src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,15 @@
     <uses-sdk android:minSdkVersion="8"/>
 
     <application>
+        <!--suppress AndroidDomInspection -->
+        <activity
+            android:name="org.acra.CrashReportDialogImpl"
```

Comment:
A generated class which extends the class annotated with @AcraDialog, if there is one.